### PR TITLE
removed ses hardcoded repos, added ses reg code

### PR DESCRIPTION
**Pull request by ellisab**
## What does this PR change?
- What was wrong with the existing code?
It was using ses hardcoded repos
- How does this PR fixes it?
Now it uses ses registration code

### CODE REVIEW

File: ses-openstack-terraform/deepsea.sh [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+sleep 1m
```
Comment (hwoarang) [inline ses-openstack-terraform/deepsea.sh]: what is this :) ? Is it unrelated?

File: ses-openstack-terraform/mon.tpl [hunk 2]
```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
-SUSEConnect -r ${regcode}
-zypper --non-interactive --quiet --gpg-auto-import-keys ar -f ${ses_base} ses_base
-zypper --non-interactive --quiet --gpg-auto-import-keys ar -f ${ses_update} ses_update
+SUSEConnect -r ${regcodeSLES}
+sleep 1
```
Comment (hwoarang) [inline ses-openstack-terraform/mon.tpl]: Why are these sleep statements necessary? Is there no better way to do this? Like a while look to check whether the product has been registered?

File: ses-openstack-terraform/ses-cluster.tf [hunk 3]
```diff
@@ -365,7 +362,7 @@ resource "openstack_compute_floatingip_associate_v2" "mon_ext_ip" {
 
 resource "openstack_blockstorage_volume_v2" "osd-blk" {
   count = "${var.osds}"
-  size  = 40
+  size  = "${var.volumeSize}"
```
Comment (hwoarang) [inline ses-openstack-terraform/ses-cluster.tf]: So this is a nice change but it is unrelated. Please move it to a separate commit

File: ses-openstack-terraform/ses-openstack [hunk 4]
```diff
@@ -87,4 +87,4 @@ fi
 
 terraform $COMMAND -var auth_url=$OS_AUTH_URL -var domain_name=$OS_USER_DOMAIN_NAME \
                     -var region_name=$OS_REGION_NAME -var project_name=$OS_PROJECT_NAME \
-                    -var user_name=$OS_USERNAME -var password=$OS_PASSWORD -var-file=openstack.tfvars
+                    -var user_name=$OS_USERNAME -var password=$OS_PASSWORD -var-file=terraform.tfvars
```
Comment (hwoarang) [inline ses-openstack-terraform/ses-openstack]: Similarly this also feels like an unrelated change to please move it to a separate commit

File: ses-openstack-terraform/deepsea.sh (hunk 1 above)
Comment (ellisab) [reply ses-openstack-terraform/deepsea.sh]: It avoids zypper locked error

File: ses-openstack-terraform/mon.tpl (hunk 2 above)
Comment (ellisab) [reply ses-openstack-terraform/mon.tpl]: It avoids a zypper locked error

File: ses-openstack-terraform/deepsea.sh (hunk 1 above)
Comment (ereslibre) [reply ses-openstack-terraform/deepsea.sh]: Can we try to do an active wait here for something that is more descriptive? I wonder how it avoids the zypper lock error.  This itself can be a problem (besides the random wait), we can hit a zypper lock error again even after passing this, right? If something else locked zypper? Maybe one option would be to even loop outside the `zypper -n in` command until it succeeds (with a limit of retries)

File: ses-openstack-terraform/mon.tpl (hunk 2 above)
Comment (ereslibre) [reply ses-openstack-terraform/mon.tpl]: Same comment as before, I agree with @hwoarang, and I'd prefer to do a more explicit conditional on what we are waiting for, or even retrying what we expect to succeed.

File: ses-openstack-terraform/deepsea.sh (hunk 1 above)
Comment (drpaneas) [reply ses-openstack-terraform/deepsea.sh]: In openQA we used to disable the packagekit as it can start out of the blue, blocking with zypper. The command we used to do that is:

```
systemctl mask packagekit; systemctl stop packagekit; while pgrep packagekitd; do sleep 1; done
```

Dunno if it helps, I just wanted to inform you about it as well :)
